packages/react/src/components/Card/CardToolbar.jsx

```diff
@@ -192,9 +193,9 @@ const CardToolbar = ({
         </OverflowMenu>
       ) : extraActions.icon ? (
         <ToolbarSVGWrapper
-          title={mergedI18n.extraActionLabel}
+          title={extraActions.iconDescription || mergedI18n.extraActionLabel}
```

Comment:
Why can't we use the translation string that is already provided? This change will bring down the test coverage and is redundant, no?

Reply:
I thought of making making it uniform by providing extraActions. but you are correct.  Let me revert it to extraActionLabel

Reply:
![image](https://user-images.githubusercontent.com/18572477/187168388-fb18cd9b-4613-40a7-a703-a46b1e3c1a92.png)

The reason behind that is if we have multiple card with multiple tooltip then its always taking the global ones. So to avoid that I have moved it into the `extraActions`

Reply:
The card is an isolated scope. You can pass in the extra action Icon description for the value of `i18n.extraActionLabel`, no?

Reply:
Unfortunately, the extraActionLabel is at the global scope. That's why we need to handle it in card-level scope.